app/views/peer_reviews/_peer_reviews_new_manager.js.jsx.erb

```diff
@@ -96,6 +96,11 @@
       return (
         <div>
           <ErrorDiv error={this.state.error} />
+          <div className="peer-review-scrollbar-div">
+            <%= j raw t('peer_review.number_per_group') %>: <input type="number"
```

Comment:
Currently it's not part of any form that's submitted, I was hoping to use JavaScript to pull out the value when a user presses the random button and transmit that via ajax.
Assuming there's not a better way and the above is to be done, can the name be left off then?